src/components/binding/binding-loader.js

```diff
@@ -28,19 +28,23 @@ genotet.BindingLoader.prototype.LOCUS_MARGIN_RATIO = .1;
 
 /**
  * Loads the binding data for a given gene and chromosome.
- * @param {string} fileName Binding file name.
+ * @param {!Array<string>} fileNames Binding file names.
  * @param {string} bedName Bed name of the bed binding track.
  * @param {string} chr ID of the chromosome.
  * @param {number=} opt_track Track # into which the data is loaded.
  * @override
  */
-genotet.BindingLoader.prototype.load = function(fileName, bedName, chr,
+genotet.BindingLoader.prototype.load = function(fileNames, bedName, chr,
                                                 opt_track) {
   var trackIndex = opt_track ? opt_track : this.data.tracks.length;
+  var isAddTrack = !opt_track;
```

Comment:
How does isAddTrack relate to the negation of track index? 

Since you are changing load to accept multiple filenames, track indices should be multiple as well. If you make them not corresponding to each other, then you are probably modifying this method in a wrong way.